<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalkRoute</class>
 <widget class="QWidget" name="WalkRoute">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="endLabel">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>140</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startEdit">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>100</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gate No.4</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="endEdit">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>140</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gate No.2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryButton">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1191</x>
     <y>450</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>园艺小镇:Gardening Town</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>480</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>国际馆:International Hall</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>510</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>中国馆:Chinese Hall</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>540</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>妫汭剧场:Guirui Theatre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>570</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>生活体验馆:Life Experience Hall</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>600</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>植物馆:Botanical Hall</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>420</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>[注]各景点名称如下：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearButton">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>180</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelStartPos">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>290</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Start :</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelEndPos">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>320</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>End :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>630</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>永宁阁:Yongning Pavilion</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NextPathButton">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>NextPath</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PrevPathButton">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>220</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PrevPath</string>
   </property>
  </widget>
  <widget class="QLabel" name="totallen">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>350</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Length :</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelRoute">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>380</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Route :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>660</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>康延路:Kangyan Road</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>690</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>妫汭湖:Guirui Lake</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>720</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1（2,4,5,6）号门:Gate No.1/2/4/5/6</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
